english/33hours/explore-outside/outside.md

```diff
@@ -3,6 +3,6 @@ do you see your surroundings. You are standing near a long two-lane highway that
 left and your right. You stand there several minutes, but see no cars at all.
 
 The motel parking lot is completely empty. It looks to be abandoned. There are no other buildings around for as far as
-you can see.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.